recover_affected_workloads.md

```diff
@@ -21,12 +21,17 @@ For example, in the case where the node fails to update its status to the
 masters the node condition will be set to `type:Ready status:Unknown`.
 
 In order to gain the workload back, reschedule it and start it all over, the
-control plane needs to know that the running node is gone and the only way to achieve
-that is by deleting the node object, which will trigger [scheduling the pods for
-deletion][2]
+control plane needs to know that the running node is gone and there are two ways to do it:
+
+- deleting the node object, which will trigger [scheduling the pods for deletion][2]
```

Comment:
> deleting the node object

Good job on updating the doc, but I think we are confusing here couple of different strategies.
- Node deletion (which was completely removed), this strategy used to delete the Node.
- ResourceDeletion - which explicitly deletes the pods of the remediated node
- OutOfServiceTaint - which implicitly causes the node's resource removal in a more gracefully way by placing a taint on the node.

ATM only the later 2 are being used.
SNR also has a third strategy`Automatic`  which basically let's the let's the remediator choose how to perform the remediation, which is something which might be worth mentioning as well.

"Under the hood" `Automatic`   just dynamically chooses OutOfServiceTaint strategy in case it's supported, otherwise it chooses ResourceDeletion, but I don't think we want to document this part.